cmd/release/docs/existing_docs/readme.go

```diff
@@ -0,0 +1,36 @@
+package existing_docs
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	. "github.com/aws/eks-distro/cmd/release/utils"
+	"os"
+)
+
+// UpdateREADME updates the README to replace release manifest from previous patch release with the new one.
+func UpdateREADME(release Release, readmePath string) error {
+	data, err := os.ReadFile(readmePath)
+	if err != nil {
+		return fmt.Errorf("failed to read README because error: %v", err)
+	}
+
+	//	Example:
```

Comment:
I wonder about wether this comment should be moved into the godoc for the method

Reply:
What godoc are you referencing?